specs/Standards/Tokens/FungibleToken.md

```diff
@@ -147,13 +163,22 @@ When called, DEX makes 2 async transfers calls to exchange corresponding tokens.
 ## Reference-level explanation
 [reference-level-explanation]: #reference-level-explanation
 
-The full implementation in Rust can be found there: https://github.com/nearprotocol/near-sdk-rs/blob/master/examples/fungible-token/src/lib.rs
+The full implementation in Rust can be found there: [fungible-token](https://github.com/nearprotocol/near-sdk-rs/blob/master/examples/fungible-token/src/lib.rs)
 
 **NOTES:**
-- All amounts, balances and allowance are limited by U128 (max value `2**128 - 1`).
+- All amounts, balances and allowance are limited by `U128` (max value `2**128 - 1`).
 - Token standard uses JSON for serialization of arguments and results.
 - Amounts in arguments and results have are serialized as Base-10 strings, e.g. `"100"`. This is done to avoid
 JSON limitation of max integer value of `2**53`.
+- The contract tracks the change in storage before and after the call. If the storage increases,
+the contract requires the caller of the contract to attach enough deposit to the function call
+to cover the storage cost.
+This is done to prevent a denial of service attack on the contract by taking all available storage.
```

Comment:
This seems a little vague.  Perhaps mention that since the cost of adding new escrow account is cheap, many accounts can be created until the contract runs out of stroage.